### PR TITLE
Add get/set to item description tooltip

### LINKED ISSUE #7007: Viewing whether a property is get-only, get-set, or set-only
**Is your feature request related to a problem? Please describe.**
Consider these three properties:
```fs
type C() =
    member __.GetOnly = 3
    member __.GetSet with get () = () and set () = ()
    member __.SetOnly with set () = ()

let c = C()
c.
```
-----------------
image
-----------------
image
-----------------
image

**Describe the solution you'd like**
Whether a given property is get-only, get-set, or set-only should be clearly be differentiated. e.g. showing `with get`, `with get, set` and `with set` after the current signature.

**Describe alternatives you've considered**
Doing nothing. An assignment will need to be tried, which is disturbing and annoying.

**Additional context**
No

### PR DESCRIPTION
**Pull request by cartermp**
Fixes #7007 by adding an additional section in the formatted tooltip text for properties that indicates if it is a getter, setter, or both.

Using this code:

```fsharp
type C() =
    member __.GetOnly = 3
    member __.GetSet with get () = () and set () = ()
    member __.SetOnly with set () = ()

let c = C()
```

Getter:

image

Setter:

image

Both:

image

### CODE REVIEW

**Comment by cartermp** [comment]
cc @Happypig375

File: src/fsharp/NicePrint.fs [hunk 1]
```diff
@@ -1405,12 +1405,22 @@ module InfoMemberPrinting =
...
+                "get, set"
+
         wordL (tagText (FSComp.SR.typeInfoProperty())) ^^
         layoutTyconRef denv pinfo.ApparentEnclosingTyconRef ^^
         SepL.dot ^^
         nameL ^^
         RightL.colon ^^
-        layoutType denv rty
+        layoutType denv rty ^^
+        wordL (tagKeyword "with") ^^
```
Comment (brettfo) [inline src/fsharp/NicePrint.fs]: `"with"` will need to be localized.

File: src/fsharp/NicePrint.fs [hunk 2]
```diff
@@ -1405,12 +1405,22 @@ module InfoMemberPrinting =
             | None -> tagProperty
             | Some vref -> tagProperty >> mkNav vref.DefinitionRange
         let nameL = DemangleOperatorNameAsLayout tagProp pinfo.PropertyName
+        let accessibilityText =
```
Comment (brettfo) [inline src/fsharp/NicePrint.fs]:
``` f#
let accessibilityTest =
    match pinfo.HasGetter, pinfo.HasSetter with
    | (true, false) -> "get"
    | (false, true) -> "set"
    | (true, true) -> "get, set"
    | (false, false) -> @"¯\_(ツ)_/¯"
```

File: src/fsharp/NicePrint.fs (hunk 1 above)
Comment (cartermp) [reply src/fsharp/NicePrint.fs]: Is that true given that it's a keyword?

Comment (brettfo) [reply src/fsharp/NicePrint.fs]: Good point, I withdraw my change request, then.

File: src/fsharp/NicePrint.fs [hunk 3]
```diff
@@ -1405,12 +1405,24 @@ module InfoMemberPrinting =
             | None -> tagProperty
             | Some vref -> tagProperty >> mkNav vref.DefinitionRange
         let nameL = DemangleOperatorNameAsLayout tagProp pinfo.PropertyName
+        let getterSetter =
+            match pinfo.HasGetter, pinfo.HasSetter with
+            | (true, false) ->
+                "with get" |> tagText |> wordL
```
Comment (saul) [inline src/fsharp/NicePrint.fs]: No longer highlighting the `with` as a keyword here?

Comment (cartermp) [reply src/fsharp/NicePrint.fs]: Whoops, my bad. Will fix.

Comment (cartermp) [reply src/fsharp/NicePrint.fs]: Done.

**Comment by cartermp** [comment]
Ready to merge when this passes. I walked through all examples in this doc to verify it all shows appropriately: https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/members/properties

**Pull request merged**